scripts/prepublish.js

```diff
@@ -15,18 +15,29 @@ const fs = require("fs");
 
 const { titleToFilename } = require("./utils");
 
-const icons = {};
+// Local helper functions
+function iconToKeyValue(icon) {
+  return `'${icon.title}':${iconToObject(icon)}`;
+}
+function iconToObject(icon) {
+  return `{title:'${icon.title}',svg:'${icon.svg}',path:'${icon.path}',source:'${icon.source.replace(/'/g, "\\'")}',hex:'${icon.hex}'}`;
```

Comment:
Maybe extract escaping to a helper function, and apply it to every embedded string?

Reply:
Good idea 👍